packages/web/components/hustler/Render.tsx

```diff
@@ -15,7 +15,7 @@ const Render = ({ itemIds }: { itemIds: BigNumber[] }) => {
   const [itemRles, setItemRles] = useState<string[]>([]);
   const [bodyRles, setBodyRles] = useState<string[]>([]);
 
-  const provider = useMemo(
+  const provider = useMemo<any>(
```

Comment:
prefer to avoid `any` types. we should be able to set this using an RPCProvider type from ethers

Reply:
Sure

Reply:
Hey @tarrencev can't seem to find this type

Reply:
Perhaps this? https://github.com/ethers-io/ethers.js/blob/master/packages/abstract-provider/src.ts/index.ts#L225